client/style.css

```diff
@@ -27,7 +27,7 @@ body {
 }
 
 h1{
-  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-family: Helvetica, Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
```

Comment:
Not sure if it's worth doing in this PR, but we probably want to spread this font choice across the whole platform, i.e. in the `html, body` rule or something similar. I assume for maintenance sake we will want to avoid small style changes on small parts, aiming instead for the top of the tree/DOM and inheriting down. Don't know if it's worth fixing for this, but just a nit

Reply:
Easy enough to add it to the top body CSS definition. 
It's done.
...oh, that looks just a little nicer, now. Good call.
HELVETICA EVERYWHERE!
(You're going to have to imagine that in the voice of "BABIES EVERYWHERE!" from the Nutrigrain video.)